exercises/allergies/.gitignore

```diff
@@ -0,0 +1,7 @@
+# Generated by Cargo
+# will have compiled files and executables
+/target/
+
+# Remove Cargo.lock from gitignore if creating an executable, leave it for libraries
```

Comment:
Question, since we know for all exercises that they are libraries, might we remove this comment? Leaving it in might give the impression that the student needs to make a choice (when the choice has already been made, I believe)